docs/DeveloperGuide.adoc

```diff
@@ -295,51 +297,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |user that takes flash card tests |select an option out of four that best matches the front of a flash card |I can check that I have understood the concept the flash card aims to impart
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |beginner user |access a help page |i can learn about the features at a glance
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `FlashCard Pro` and the *Actor* is the `learner`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Create a deck of cards
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. Learner creates a new empty deck and specifies the name of the deck of cards.
+2. Learner adds individual cards to the deck and specifies the content of the front and back of the cards.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. FlashCard Pro detects deck name provided by learner is identical to an existing deck of cards in the library.
+** 1a1. FlashCard Pro informs the learner that deck creation is unsuccessful.
+** 1a2. Learner creates a new deck with another unique deck name.
+** Steps 1a1-1a2 are repeated until the data entered are correct.
+** Use case resumes from step 2.
+
+* 2a. FlashCard Pro detects no front and/or back content specified by the learner.
+** 2a1. FlashCard Pro informs the learner that card creation is unsuccessful.
+** 2a2. Learner creates a new card with the correct front and/or back content.
+** Steps 2a1-2a2 are repeated until the format of the command entered is correct.
 +
-Use case ends.
+(Was wondering if FlashCard Pro should create the deck, but with a numbering like untitled_1, untitled_2 etc)
+
+
+[discrete]
+=== Use case: Start a untimed test
+
+*MSS*
 
-* 3a. The given index is invalid.
+1. Learner starts untimed test with a deck of cards by specifying deck name.
+2. FlashCard Pro begins test.
+3. FlashCard Pro flashes text on the front of a flash card to learner.
+4. FlashCard Pro waits for learner's answer.
+5. Learner enters answer.
+6. FlashCard Pro matches answer with text on back of flash card.
+7. FlashCard Pro informs learner about correctness of the learner's answer.
+8. FlashCard Pro displays the correct answer, the text on the back of the flash card.
```

Comment:
For this use case, maybe you guys can consider an extension where the user indicates that he or she wishes to quit the untimed test session. This extension can happen at any step.